remote-api/src/test/java/org/alfresco/repo/web/scripts/person/PersonServiceTest.java

```diff
@@ -1,921 +1,968 @@
+                "myJobTitle", "myEmail", "myBio", "images/avatar.jpg", 0, 409);
+    }
+    
+    public void testCreatePersonMissingUserName() throws Exception
+    {
+        // Create a new person with userName == null (user name missing)
+        createPerson(null, "myTitle", "myFirstName", "myLastName", "myOrganisation",
+                        "myJobTitle", "firstName.lastName@email.com", "myBio", "images/avatar.jpg", 0,
+                        Status.STATUS_BAD_REQUEST);        
+        
+        // Create a new person with userName == "" (user name is blank)
+        createPerson("", "myTitle", "myFirstName", "myLastName", "myOrganisation",
+                        "myJobTitle", "firstName.lastName@email.com", "myBio", "images/avatar.jpg", 0,
+                        Status.STATUS_BAD_REQUEST);        
+    }
+    
+    public void testCreatePersonMissingFirstName() throws Exception
+    {
+        String userName  = RandomStringUtils.randomNumeric(6);
+                
+        // Create a new person with firstName == null (first name missing)
+        createPerson(userName, "myTitle", null, "myLastName", "myOrganisation",
+                        "myJobTitle", "firstName.lastName@email.com", "myBio", "images/avatar.jpg", 0,
+                        Status.STATUS_BAD_REQUEST);        
+        
+        // Create a new person with firstName == "" (first name is blank)
+        createPerson(userName, "myTitle", "", "myLastName", "myOrganisation",
+                        "myJobTitle", "firstName.lastName@email.com", "myBio", "images/avatar.jpg", 0,
+                        Status.STATUS_BAD_REQUEST);        
+    }
+
+    public void testUserNameCaseSensitivityCQ() throws Exception
+    {
+        String upperCaseUserName = "PersonServiceTest.MixedCaseUser";
+        String lowerCaseUserName = upperCaseUserName.toLowerCase();
+        // Create a new person
+
+        String currentUser = this.authenticationComponent.getCurrentUserName();
+        try
+        {
+            /**
+             *  simulate cloud with lower case user names
+             */
+            createPerson(lowerCaseUserName, "myTitle", "myFirstName", "myLastName", "myOrganisation",
+                    "myJobTitle", "firstName.lastName@email.com", "myBio", "images/avatar.jpg", 0,
+                    Status.STATUS_OK);
+
+            String adminUser = this.authenticationComponent.getSystemUserName();
+            this.authenticationComponent.setCurrentUser(adminUser);
+            personService.setCreateMissingPeople(false);
+
+            //try with canned query
+            String filter = "PerSOnSerVIceTest.MixEDCasEUseR";
+            Response response = sendRequest(new GetRequest(URL_PEOPLE + "?filter=" + filter), 200);
+            JSONObject res = new JSONObject(response.getContentAsString());
+            int peopleFound = res.getJSONArray("people").length();
+            assertTrue("No people found", peopleFound > 0);
```

Comment:
Let's apply [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). This repeated assertion block should be refactored into its own method.